tycho-its/src/test/java/org/eclipse/tycho/test/p2Repository/DownloadVerifyNoDigestAlgoTest.java

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.tycho.test.p2Repository;
+
+import static org.junit.Assert.assertFalse;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import org.apache.maven.it.Verifier;
+import org.eclipse.tycho.test.AbstractTychoIntegrationTest;
+import org.junit.Test;
+
+public class DownloadVerifyNoDigestAlgoTest extends AbstractTychoIntegrationTest {
+    @Test
+    public void test() throws Exception {
+        Verifier verifier = getVerifier("p2Repository.downloadVerifyNoDigestAlgo", false);
+        verifier.setLogFileName("p2Repository.downloadVerifyNoDigestAlgo.log");
+        verifier.executeGoal("verify");
+        verifier.verifyErrorFreeLog();
+        String logContents = Files.readString(Paths.get("p2Repository.downloadVerifyNoDigestAlgo.log"));
+        assertFalse(logContents.contains("No digest algorithm is available to verify download"));

```

Comment:
`AbstractTychoIntegrationTest` already contains some helper methods to check if log messages are (not) there.